docs/guide/acquiring_data/jsoc.rst

```diff
@@ -0,0 +1,295 @@
+into seconds.
+
+Constructing complex queries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Complex queries can be built using OR operators.
+
+Let's look for 2 dfifferent series data at the same time:
+
+    >>> res = Fido.search(a.jsoc.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00'),
+    					  a.jsoc.Series('hmi.v_45s') | a.jsoc.Series('aia.lev1_euv_12s'))
+
+The two series names are joined together by the operator "|".
+This is the ``OR`` operator.  Think of the above query as setting a set
+of conditions which get passed to the JSOC.  Let's say you want all the
+hmi.v_45s data from two separate days:
+
+    >>> res = Fido.search(a.jsoc.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00') | 
+                          a.jsoc.Time('2014-01-02T00:00:00', '2014-01-02T01:00:00'),
+                          a.jsoc.Series('hmi.v_45s'))
+
+Each of the arguments in this query style can be thought of as
+setting conditions that the returned records must satisfy.
+
+It should be noted that ``AND`` operator is supported by some of the attributes only. The attributes which
+support "&" are ``PrimeKey()``, ``Segment()``. Using "&" with any other attributes will throw an error.
+
+Downloading data
+----------------
+
+To download the files located by ``Fido.search()``, you can download them by ``Fido.fetch()``:
+
+	>>> downloaded_files = Fido.fetch(res)
+
+Using JSOCClient for complex usage
+----------------------------------
+
+Fido interface uses JSOCClient in its backend, which combines the last 2 stages the JSOC process into
+one. You can directly use the JSOC Client to make queries, instead of the FIDO Client. This will allow you
+to separate the 3 stages of the JSOC process, and perform it individually, hence allowing a greater
+control over the whole process.
+
+Setup
+^^^^^
+
+SunPy's JSOC module is in ``sunpy.net``.  It can be imported as follows:
+
+    >>> from sunpy.net import jsoc
+    >>> client = jsoc.JSOCClient()
+
+This creates your client object.
+
+Making a query
+^^^^^^^^^^^^^^
+
+Querying JSOC using the JSOCCLient is completely similar to what we were doing with Fido.
+
+	>>> from sunpy.net import attrs as a
+	>>> res = client.search(a.jsoc.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00'), a.jsoc.Series('hmi.v_45s'))
+
+Apart from the function name, everything is same. You need to pass the same values in the ``JSOCClient.search()``
+as you did in ``Fido.search()``. Complex queries can be built in a similar way, and all other things are same.
+
+Staging the request
+^^^^^^^^^^^^^^^^^^^
+
+JSOC is a 3-stage process, and after getting the query results, we need to stage a request for the data to be
+downloaded. Only then, can we download them. The download request can be staged like this:
+
+	>>> requests = client.request_data(res)
+	>>> print(requests)
+
+	<ExportRequest id="JSOC_20170713_1461", status=0>
+
+The function ``request_data()`` stages the request. It returns a ``drms.ExportRequest`` object, which has many
```

Comment:
single backticks here, if you add drms to the intersphinx links here: https://github.com/sunpy/sunpy/blob/master/docs/conf.py#L82 it should auto-link into the drms documentation.